include/sway/desktop/idle_inhibit_v1.h

```diff
@@ -22,6 +22,7 @@ struct sway_idle_inhibit_manager_v1 {
 
 struct sway_idle_inhibitor_v1 {
 	struct sway_idle_inhibit_manager_v1 *manager;
+	struct wlr_idle_inhibitor_v1 *wlr_inhibitor;
 	struct sway_view *view;
```

Comment:
Is this field still used?

Reply:
Yes, it is used for manual/"user" inhibitors made with the `inhibit_idle` command. It is only accessed if `inhibitor->mode` is not `INHIBIT_IDLE_APPLICATION`.